### ISSUE TITLE
Design feedbacks

### ISSUE THREAD

**Issue by josephj** [trust2065/orchard#28]
Though I don't have the access to the Invision design spec, I still feel several details are not implemented correctly.

**Their Design**
![image](https://user-images.githubusercontent.com/136648/61988503-a3998700-b054-11e9-9d0c-18c7cda737a6.png)

**Your Implementation**
![image](https://user-images.githubusercontent.com/136648/61988518-e196ab00-b054-11e9-8a4d-f6cdbd136a50.png)

* Page margin is not correct
* Title font-weight is not correct, should be `font-weight-light`. (You need to have `300` from Google Font)
* Title border-bottom color is not correct
* Content font-weight is not correct, should be `font-weight-light`.
* Content font color is not correct, should not be `white`
* Be mindful title and content font-weight is not correct, should be `semibold` (`700`) rather than `bold`. Can you create `font-weight-semibold` from the bootstrap setting?

**Comment by trust2065** [comment]
![image](https://user-images.githubusercontent.com/13488449/62069825-7d563000-b27c-11e9-8186-bbcac074b745.png)

**Comment by trust2065** [comment]
> Page margin is not correct
I've never think of change container's width, I thought what it provide is the best.
But now I feel change it is not a big deal. We can have different container width in different project and they all using `Bootstrap`!

**Comment by josephj** [comment]
You don't need to worry `16px`, right? So maybe just create `font-size-lg`. (Bootstrap uses `sm`, `md`, `lg`, and `xl`)

UI design is always customized. Default is just a baseline for you to tweak.

**Comment by trust2065** [comment]
**1. Spacing** 

Spacing between section1 images is solved by setting $space: 0.7rem, but it also make spacing on other different element difficult.

![image](https://user-images.githubusercontent.com/13488449/62184470-93114580-b3a1-11e9-9ea1-16bd1fa19a6b.png)

Now I'm not sure that is a good way to use $space.
I understand I'm still producing question, but with many various spacing, it's pretty unfriendly to use $space to handle all spacing.

Unless I set $spacing to 0.1rem, problem solved but it's not make sense!

---

**2. color doesn't match**
Even use color in invision, it doesn't match.
In real case, I would ask them which color they want.
![image](https://user-images.githubusercontent.com/13488449/62184643-4417e000-b3a2-11e9-8f77-67110964890c.png)


---
**3. make bottom-border thick**
**solution 1, stick with bootstrap class:** 
```
.border-width-2 {
  border-bottom-width: 2px !important; // in order to overwrite !important in bootstrap
}

.section {
  &Image {
    @extend .border-danger;
    @extend .border-bottom;  //translate to `border-bottom: 1px solid #dee2e6 !important;`
    @extend .border-width-2;
  }
```
**solution 2**
```
.section {
  &Image {
    border: 0px solid;
    border-bottom: 2px;
    @extend .border-danger;
  }
```

I don't like to use !important :(
